### ISSUE TITLE
dualmode zil incompatible with nicehash

### ISSUE THREAD

**Issue by mew905** [Lolliedieb/lolMiner-releases#321]
using the --dualmode zil --dualstratum commands I simply can't get a connection to daggerhashimoto. Tried with and without the stratum+tcp:// bit, different servers, and even testing for connectivity issues on my PC and am about 80% confident the issue lies within the application.

**Comment by jgonzis** [comment]
> using the --dualmode zil --dualstratum commands I simply can't get a connection to daggerhashimoto. Tried with and without the stratum+tcp:// bit, different servers, and even testing for connectivity issues on my PC and am about 80% confident the issue lies within the application.

Only add:
 --ethstratum ETHPROXY,ETHV1

**Comment by mew905** [comment]
> > using the --dualmode zil --dualstratum commands I simply can't get a connection to daggerhashimoto. Tried with and without the stratum+tcp:// bit, different servers, and even testing for connectivity issues on my PC and am about 80% confident the issue lies within the application.
> 
> Only add:
> --ethstratum ETHPROXY,ETHV1
Nope. Didn't work
What do each of those do? I'm having a hard time finding any documentation about lolminer arguments
```
Lost connection to stratum server daggerhashimoto.usa.nicehash.com:3353 or server not reachable. 
Trying to connect in 1 second 
Muting ZIL connection: currently no ZIL work. 
------------------------------------------------------- 
         Generating light cache for epoch 408 
------------------------------------------------------- 
Start Mining... 
Connected to daggerhashimoto.usa.nicehash.com:3353  (TLS disabled)
Set Ethash stratum mode: Ethereum Proxy
Lost connection to stratum server daggerhashimoto.usa.nicehash.com:3353 or server not reachable. 
Trying to connect in 1 second 
Connected to daggerhashimoto.usa.nicehash.com:3353  (TLS disabled)
Set Ethash stratum mode: Ethereum Proxy
Lost connection to stratum server daggerhashimoto.usa.nicehash.com:3353 or server not reachable. 
Trying to connect in 1 second 
------------------------------------------------------- 
     Done (2953 ms), size of new DAG: 4288 MByte 
------------------------------------------------------- 
Start DAG gen on GPU 0 (normal mode) 
Start DAG gen on GPU 3 (normal mode) 
Start DAG gen on GPU 1 (normal mode) 
Start DAG gen on GPU 2 (normal mode) 
Connected to daggerhashimoto.usa.nicehash.com:3353  (TLS disabled)
Set Ethash stratum mode: Ethereum Proxy
Lost connection to stratum server daggerhashimoto.usa.nicehash.com:3353 or server not reachable. 
Trying to connect in 1 second 
Connected to daggerhashimoto.usa.nicehash.com:3353  (TLS disabled)
Set Ethash stratum mode: Ethereum Proxy
Lost connection to stratum server daggerhashimoto.usa.nicehash.com:3353 or server not reachable. 
Trying to connect in 1 second 
```

**Comment by mew905** [comment]
HiveOS Miner Config reports as such
```--algo ETHASH
--pool us-west.ezil.me:4444
--user [eth addy].[zil addy]
--pass x
--pool us-west.ezil.me:14444
--user [eth addy].[zil addy]
--pass x
--worker Mining_Rig
--ethstratum ETHPROXY,ETHV1
--devices 0,1,2,3
--dualmode zil
--dualstratum [nicehash wallet removed].[nicehash worker ID removed]@stratum+tcp://daggerhashimoto.usa.nicehash.com:3353```

**Comment by jgonzis** [comment]
Only to check, are you in 1.24a?.

The version 1.20 didn't accept the --ethstratum ETHPROXY,ETHV1

To see that info I always go to the miner and lolMiner -h

As I see for your log it is trying to connect to nicehash pool with the bad parameters,  he should connect in nicehash mode and not ETHPROXY mode

**Comment by mew905** [comment]
> Only to check, are you in 1.24a?.
> 
> The version 1.20 didn't accept the --ethstratum ETHPROXY,ETHV1
> 
> To see that info I always go to the miner and lolMiner -h
> 
> As I see for your log it is trying to connect to nicehash pool with the bad parameters, he should connect in nicehash mode and not ETHPROXY mode

I'm using HiveOS and it's set to "The Latest" (for LoLminer, it does list 1.24a). Running with -h parameter is slow and clumsy via the webUI, though I did find LOLminer's readme in my Windows PC's Nicehash folder.
I tried using ETHV1 and I get an error saying the pool responded with an Ethproxy response.

I'm assuming this is due to a conflict between Nicehash's ETHV1 stratum and ezil's ETHPROXY stratum.

I dont doubt it'd play nice with a different ethpool (like ethermine) but if I'm not using nicehash, I'm using ezil.
